tests/regression/gmail/test_chrome_gmail_ft_type_in_reply_field_100_txt_delay_100ms.sikuli/test_chrome_gmail_ft_type_in_reply_field_100_txt_delay_100ms.py

```diff
@@ -0,0 +1,83 @@
+# if you are putting your test script folders under {git project folder}/tests/, it will work fine.
+# otherwise, you either add it to system path before you run or hard coded it in here.
+
+INPUT_LIB_PATH = sys.argv[1]
+sys.path.append(INPUT_LIB_PATH)
+
+import os
+import basecase
+import gmail
+
+import string
+import shutil
+import browser
+import time
+
+
+class Case(basecase.SikuliInputLatencyCase):
+
+    def run(self):
+        # Disable Sikuli action and info log
+        self.common.infolog_enable(False)
+        Settings.MoveMouseDelay = 0
+        self.common.set_type_delay(0.08)
+
+        # Prepare
+        app = gmail.gMail()
+        sample1_fp = os.path.join(self.INPUT_IMG_SAMPLE_DIR_PATH, self.INPUT_IMG_OUTPUT_SAMPLE_1_NAME)
+        sample2_fp = os.path.join(self.INPUT_IMG_SAMPLE_DIR_PATH, self.INPUT_IMG_OUTPUT_SAMPLE_1_NAME.replace('sample_1', 'sample_2'))
+        capture_width = int(self.INPUT_RECORD_WIDTH)
+        capture_height = int(self.INPUT_RECORD_HEIGHT)
+
+        # Launch browser
+        my_browser = browser.Chrome()
+
+        # Access link and wait
+        my_browser.clickBar()
+        my_browser.enterLink(self.INPUT_TEST_TARGET)
+        app.wait_for_loaded()
+
+        # Wait for stable
+        sleep(2)
+
+        # PRE ACTIONS
+        app.click_first_mail()
+        sleep(2)
+        app.click_reply_btn()
+        sleep(2)
+
+        # Customized Region
+        customized_region_name_start = 'start'
+        customized_region_name_end = 'end'
+
+        # part region of search suggestion list
+        _, reply_btn_region = app.wait_for_component_display(app.GMAIL_TYPE_FOR_REPLY, similarity=0.8)
+        compare_area = self.tuning_region(reply_btn_region, x_offset=-10, y_offset=30, w_offset=650, h_offset=15)
+        self.set_override_region_settings(customized_region_name_start, compare_area)
+        self.set_override_region_settings(customized_region_name_end, compare_area)
+
+        t1 = time.time()
+        capimg1 = capture(0, 0, capture_width, capture_height)
+
+        char_len = 100
+        sample_str = string.letters + "\n" + string.letters[::-1]
+        sample_str.replace("t", "Q").replace("i", "Q").replace("j", "Q").replace("l", "Q")\
```

Comment:
Please Refer `https://stackoverflow.com/questions/3411771/multiple-character-replace-with-python` for the reason I chose to replace characters in this way.